Examples/Examples-iOS/IGListKitExamples.xcodeproj/xcshareddata/xcschemes/IGListKitTodayExample.xcscheme

```diff
@@ -43,13 +43,23 @@
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES">
       <Testables>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "95F7F9101DE5006B00A64FEE"
+               BuildableName = "IGListKitExamples-UITests.xctest"
+               BlueprintName = "IGListKitExamples-UITests"
+               ReferencedContainer = "container:IGListKitExamples.xcodeproj">
+            </BuildableReference>
+         </TestableReference>
       </Testables>
       <MacroExpansion>
          <BuildableReference
             BuildableIdentifier = "primary"
-            BlueprintIdentifier = "986FB72E1DBBAD8600A65C18"
-            BuildableName = "IGListKitTodayExample.appex"
-            BlueprintName = "IGListKitTodayExample"
```

Comment:
@jessesquires Weird that only the scheme from Today extension has these changes.
I just created a clean UI Test target, branched from master, and the same thing happened. 
Both Message and Today extensions build, run, and launch the UI Tests successfully.
Do you have any suggestions?